README.md

```diff
@@ -1,4 +1,113 @@
-# Finetuner
+<p align="center">
+<img src="https://github.com/jina-ai/finetuner/blob/main/docs/_static/finetuner-logo-ani.svg?raw=true" alt="Finetuner logo: Finetuner helps you to create experiments in order to improve embeddings on search tasks. It accompanies Jina to deliver the last mile of performance-tuning for neural search applications." width="150px">
+</p>
 
-This package implements a client API for creating and running fine-tuning experiments.
 
+<p align="center">
+<b>Fine-tuning embeddings on domain specific data for better performance on neural search tasks.</b>
+</p>
+
+<p align=center>
+<a href="https://pypi.org/project/finetuner/"><img src="https://img.shields.io/badge/Python-3.9%2B-blue alt="Python 3.9" title="Finetuner supports Python 3.9 and above"></a>
+<a href="https://slack.jina.ai"><img src="https://img.shields.io/badge/Slack-2.2k%2B-blueviolet?logo=slack&amp;logoColor=white"></a>
+</p>
+
+<!-- start elevator-pitch -->
+
+Fine-tuning deep neural networks (DNNs) significantly improves performance on domain specific neural search tasks.
+However, fine-tuning for neural search is not trivial, as it requires a combination of expertise in ML and Information Retrieval.
+Finetuner makes finetuning simple and fast by handling all related complexity and infrastructure in the cloud. With Finetuner, you can easily make models more perfomant and production ready.
+
+📈**Performance boost**: Finetuner significantly increases the performance of pretrained models on domain specific neural search applications.
+
+🔱 **Simple yet powerful**: Interacting with Finetuner is simple and seamless, and also supports rich features such as
+siamese/triplet loss, metric learning, self-supervised pretraining, layer pruning, weights freezing, dimensionality reduction, and much more.
+
+☁ **Finetune in the cloud**: Finetuner runs your fine-tuning jobs on [Jina Cloud](https://github.com/jina-ai/jcloud). You never have to worry about provisioning (cloud) resources! Finetuner handles all related complexity and infrastructure.
+
+<!-- end elevator-pitch -->
+
+## What is the purpose of Finetuner?
+
+Finetuner enables performance gains on domain specific neural search tasks by allowing you to fine-tune models in the cloud. We have conducted experiments on three neural search tasks in different domains to illustrate these performance improvements.
```

Comment:
can we say experiments on varies tasks such as ..., not mention three tasks? we're actually experimenting on a lot of tasks. For me it's like we